.pre-commit-config.yaml

```diff
@@ -1,24 +1,20 @@
 exclude: '\.snap$'
 repos:
--   repo: git://github.com/pre-commit/mirrors-autopep8
-    sha: v1.3.3
+-   repo: https://github.com/ambv/black
+    rev: b07ca2b7d5d03f5d890c1ce5f19b41065ba815c4
     hooks:
-    -   id: autopep8
+    - id: black
+      args: [--line-length=88, --safe]
```

Comment:
`--line-length=88, --safe` are the defaults.

Reply:
just being explicit per the example configuration in Black